cmd/server.go

```diff
@@ -40,7 +40,11 @@ var serverCmd = &cobra.Command{
 		natsurl := strings.Join(hosts, ",")
 
 		var natsCredentials nats.Option
-		companyID, _ := cmd.Flags().GetString("company-id")
+
+		companyIDOpt, _ := cmd.Flags().GetString("company-id")
+		companyIDs := strings.Split(companyIDOpt, ",")
+
+		companyName := "unknown"
```

Comment:
Does this need some random thing with it so it's not accidentally shared as a consumer if someone spins this up the first time??

Reply:
They'd have to have no credential to get into this state or have the keys to adjust their credential and remove the name field in the JWT. It should be ok. 

Running it locally shouldn't be a big deal if we want to do that unauthed.